web/turkey/survey/static/survey/js/auditors/keypresses_total.js

```diff
@@ -0,0 +1,25 @@
+var AuditorKeypressesTotal = {
+    keypresses_total: 0,
+    log_keypress_event: function (e) {
+        // e.stopPropagation();
```

Comment:
I'm more confused about stopPropagation now, seeing as how some are commented out and some still exist